js/source/scripts.js

```diff
@@ -10,27 +10,128 @@
 // wrapping it with an "anonymous closure". See:
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
-(function (Drupal, $, window) {
+ 
+  // Debounce https://davidwalsh.name/javascript-debounce-function
+ (function ($) {
+
+  function debounce(func, wait, immediate) {
+    var timeout;
+    return function() {
+      var context = this, args = arguments;
+      var later = function() {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      var callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
+    };
+  };
 
   // To understand behaviors, see https://www.drupal.org/node/2269515
   Drupal.behaviors.basic = {
     attach: function (context, settings) {
 
       // Execute code once the DOM is ready. $(document).ready() not required within Drupal.behaviors.
-
       $(window).load(function () {
```

Comment:
a window.load inside of a Drupal.behavior.attach function is not allowed. Please see: https://www.drupal.org/docs/8/api/javascript-api/javascript-api-overview

The TLDR; version is that the attach method is a fancy Drupal wrapper for `$(document).ready(function () {` that is also called when ajax calls are finished. If it is possible to run on DOMReady instead of on window load you can simply remove the load wrapper. If you need all of the items to load to calculate widths etc, you can move this outside of the Drupal.behavior wrapping and simply keep it on its own.